be/src/storage/tablet_manager.cpp

```diff
@@ -1762,4 +1768,16 @@ std::unordered_map<TTabletId, vector<pair<uint32_t, uint32_t>>> TabletManager::g
     return ret;
 }
 
+void TabletManager::_add_shutdown_tablet_unlocked(int64_t tablet_id, DroppedTabletInfo&& drop_info) {
+    auto iter = _shutdown_tablets.find(tablet_id);
+    if (iter != _shutdown_tablets.end() && (iter->second).tablet != nullptr) {
```

Comment:
also need erase from map if tablet is nullptr?

Reply:
I think we need it. Here we still use std::map:emplace to add a new shutdown tablet into the map. This api will reject the new shutdown tablet because this nullptr has occupy the coresspoding key(tabletid) in this map.

Reply:
updated